docs-ui/components/checkboxFancy.stories.js

```diff
@@ -1,24 +1,28 @@
 import React from 'react';
-import {storiesOf} from '@storybook/react';
 import {withInfo} from '@storybook/addon-info';
 import {number, boolean} from '@storybook/addon-knobs';
 import styled from '@emotion/styled';
 
 import CheckboxFancy from 'app/components/checkboxFancy/checkboxFancy';
 
-storiesOf('Style|Icons', module).add(
-  'CheckboxFancy',
-  withInfo('A fancy looking checkbox')(() => {
-    return (
-      <Container>
-        <CheckboxFancy
-          size={`${number('Size', 100)}px`}
-          isChecked={boolean('Checked', true)}
-        />
-      </Container>
-    );
-  })
-);
+export default {
+  title: 'Style/Icons',
+};
+
+export const _CheckboxFancy = withInfo('A fancy looking checkbox')(() => {
```

Comment:
CheckboxFancyComponent would not be a better name as we usually use _ for non-used parameters, etc?

Reply:
This code is all auto generated, but yes I would agree